### DISCUSSION TITLE
DIPY

### DISCUSSION THREAD

**Discussion by Enoch-Oluwanifemi** [dipy/dipy#3077] [#introduceyourself]
Please i want to know if I can start writing my proposals for your organization

**Comment by skoudoro** [comment]
Yes, you can @Enoch-Oluwanifemi.

Also do not forget that Python Organisation require you to do a first contribution. Otherwise, your application will not be fully take in consideration. It is a way to go familiar with the codebase.

Feel free to share your draft. We review one time one draft per candidate

**Reply by Enoch-Oluwanifemi** [reply]
First contribution. What am i too do?
